atom/browser/api/atom_api_menu_mac.mm

```diff
@@ -16,6 +16,12 @@
 
 using content::BrowserThread;
 
+namespace {
+
+static NSMenu* applicationMenu_ = nil;
```

Comment:
We shouldn't need `static` here while in the namespace right?

Reply:
@MarshallOfSound I'd prefer having the `static` keyword there so it won't conflict with other `applicationMenu_` in other files in the future due to redefinition.

Here are some other usages of this keyword that I've found:

```
./atom/browser/mac/atom_application_delegate.mm
  22,1: static base::mac::ScopedObjCClassSwizzler* g_swizzle_imk_input_session;

./atom/browser/api/atom_api_power_monitor_mac.mm
  64,1: static MacLockMonitor* g_lock_monitor = nil;

./atom/browser/ui/cocoa/atom_touch_bar.mm
  15,1: static NSTouchBarItemIdentifier ButtonIdentifier =
  17,1: static NSTouchBarItemIdentifier ColorPickerIdentifier =
  19,1: static NSTouchBarItemIdentifier GroupIdentifier =
  21,1: static NSTouchBarItemIdentifier LabelIdentifier =
  23,1: static NSTouchBarItemIdentifier PopoverIdentifier =
  25,1: static NSTouchBarItemIdentifier SliderIdentifier =
  27,1: static NSTouchBarItemIdentifier SegmentedControlIdentifier =
  29,1: static NSTouchBarItemIdentifier ScrubberIdentifier =
  32,1: static NSString* const TextScrubberItemIdentifier = @"scrubber.text.item";
  33,1: static NSString* const ImageScrubberItemIdentifier = @"scrubber.image.item";

./atom/browser/ui/cocoa/atom_menu_controller.mm
  62,1: static base::scoped_nsobject<NSMenuItem> recentDocumentsMenuItem_;
  65,1: static base::scoped_nsobject<NSMenu> recentDocumentsMenuSwap_;
```